node/src/chain_spec.rs

```diff
@@ -35,157 +35,161 @@ pub type ChainSpec = sc_service::GenericChainSpec<GenesisConfig>;
+        AccountPublic: From<<TPublic::Pair as Pair>::Public>
 {
-	AccountPublic::from(get_from_seed::<TPublic>(seed)).into_account()
+        AccountPublic::from(get_from_seed::<TPublic>(seed)).into_account()
 }
 
 /// Generate an Aura authority key.
 pub fn authority_keys_from_seed(s: &str) -> (AuraId, GrandpaId) {
-	(
-		get_from_seed::<AuraId>(s),
-		get_from_seed::<GrandpaId>(s),
-	)
+        (
+                get_from_seed::<AuraId>(s),
+                get_from_seed::<GrandpaId>(s),
+        )
 }
 
 pub fn development_config() -> Result<ChainSpec, String> {
-	let wasm_binary = WASM_BINARY.ok_or("Development wasm binary not available".to_string())?;
-
-	Ok(ChainSpec::from_genesis(
-		// Name
-		"Development",
-		// ID
-		"dev",
-		ChainType::Development,
-		move || testnet_genesis(
-			wasm_binary,
-			// Initial PoA authorities
-			vec![
-				authority_keys_from_seed("Alice"),
-			],
-			// Sudo account
-			get_account_id_from_seed::<sr25519::Public>("Alice"),
-			// Pre-funded accounts
-			vec![
-				get_account_id_from_seed::<sr25519::Public>("Alice"),
-				get_account_id_from_seed::<sr25519::Public>("Bob"),
-				get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
-				get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
-			],
-			true,
-		),
-		// Bootnodes
-		vec![],
-		// Telemetry
-		None,
-		// Protocol ID
-		None,
-		// Properties
-		None,
-		// Extensions
-		None,
-	))
+        let wasm_binary = WASM_BINARY.ok_or("Development wasm binary not available".to_string())?;
+
+        Ok(ChainSpec::from_genesis(
+                // Name
+                "Development",
+                // ID
+                "dev",
+                ChainType::Development,
+                move || testnet_genesis(
+                        wasm_binary,
+                        // Initial PoA authorities
+                        vec![
+                                authority_keys_from_seed("Alice"),
+                        ],
+                        // Sudo account
+                        get_account_id_from_seed::<sr25519::Public>("Alice"),
+                        // Pre-funded accounts
+                        vec![
+                                get_account_id_from_seed::<sr25519::Public>("Alice"),
+                                get_account_id_from_seed::<sr25519::Public>("Bob"),
+                                get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
+                                get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
+                        ],
+                        true,
+                ),
+                // Bootnodes
+                vec![],
+                // Telemetry
+                None,
+                // Protocol ID
+                None,
+                // Properties
+                None,
+                // Extensions
+                None,
+        ))
 }
 
 pub fn local_testnet_config() -> Result<ChainSpec, String> {
```

Comment:
We use tabs for indentation.

Reply:
None of the files are formatted properly. I wanted to run the formatter, but that would mix my changes with the formatting changes. Once this is merged, we can run the formatter which will fix these problems.

Reply:
Please fix the formatting before asking for review, otherwise it makes it harder to review and introduce noise into gitlogs/blame